nms/packages/magmalte/app/views/subscriber/SubscriberUtils.js

```diff
@@ -59,6 +61,7 @@ export const SUBSCRIBER_EXPORT_COLUMNS = [
   {title: 'Auth Key', field: 'auth_key'},
   {title: 'Auth OPC', field: 'auth_opc'},
   {title: 'Service', field: 'state'},
+  {title: 'Network Type', field: 'allowed_nw_types'},
```

Comment:
```suggestion
  {title: 'Core Network Type', field: 'allowed_core_nw_types'},
```

Reply:
I changed the title to `Core Network Type`, but the field is `allowed_nw_types` according to the MagmaAPIBindings.